app/src/main/java/com/andrew/tasky/agenda/data/event/EventMapperFunctions.kt

```diff
@@ -0,0 +1,22 @@
+package com.andrew.tasky.agenda.data.event
+
+import com.andrew.tasky.agenda.data.util.ReminderTimeConversion
+import com.andrew.tasky.agenda.data.util.localDateTimeToZonedEpochMilli
+import com.andrew.tasky.agenda.domain.models.AgendaItem
+
+fun AgendaItem.Event.toCreateEventRequest(): CreateEventRequest {
+    return CreateEventRequest(
+        id = id,
+        title = title,
+        description = description,
+        from = localDateTimeToZonedEpochMilli(startDateAndTime),
+        to = localDateTimeToZonedEpochMilli(endDateAndTime),
+        remindAt = ReminderTimeConversion.toEpochMilli(
+            reminderTime = reminderTime,
+            startLocalDateTime = startDateAndTime
+        ),
+        attendeeIds = attendees?.map {
```

Comment:
Why can `attendees` be null here? If there are no attendees, I'd simply use an empty list because that says exactly that. I'd use null values for all other types where it's not clear that a default value indicates a non-existent value like for Strings